reoptjl/models.py

```diff
@@ -6420,6 +6693,12 @@ class AbsorptionChillerInputs(BaseModel, models.Model):
         'hot_water'
     ))
 
+    HEATING_LOAD_INPUT = models.TextChoices('HEATING_LOAD_INPUT', (
+        'DomesitHotWater',
```

Comment:
fixed this spelling in https://github.com/NREL/REopt_API/pull/583/commits/abf56a1e8cf0c94e1e24797785f88290bf010b69

Reply:
Thanks for catching this and the changelog update!